examples/basic.rs

```diff
@@ -16,10 +16,10 @@ use cueball::resolver::{BackendAddedMsg, BackendMsg, Resolver};
 #[derive(Debug)]
 pub struct DummyConnection {
     addr: SocketAddr,
-    connected: bool
+    connected: bool,
```

Comment:
Are these last-field trailing commas suggested by clippy? I guess it makes all entries consistent?

Reply:
They're added by `rustfmt` and they're the devil I say (while shaking fist in the air)! I do hate that it does this and in the future this particular behavior will be configurable, but for now it can't be controlled if you want to run `rustfmt`. I think it's safe to have them or not and I really prefer not. I can see how it makes it convenient when you inevitably go to add another field to the struct, but I would prefer a style using preceding commas to the dangling commas at the end.